src/models/spaces/charts/chart-data-set.ts

```diff
@@ -0,0 +1,144 @@
+import { types, Instance } from "mobx-state-tree";
+
+export interface Color {
+  cssName: string;
+  name: string;
+  hex: string;
+}
+
+interface XYPoint {
+  x: number;
+  y: number;
+}
+// As defined in vars.sass, no easy way to get these variables in here
+// so reluctantly duplicating the definitions.
```

Comment:
I was hoping we could pull these out of the sass file.  Oh well.  Quick note, I started yanking a few generic colors from vars.sass and filling in spec colors defined here:
https://fh8yk2.axshare.com/#g=1&p=4_connectedbio_-_specs__v1_3_
If colors are duplicated for the chart in code, then we need to remember to pull some UI spec colors out for use in the graph (this can probably wait until Michael has a chance to review and offer color input).

Reply:
It looks like it should be possible to export the sass variables using the :export directive: https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript

You can export variables from any style sheet: https://www.bluematador.com/blog/how-to-share-variables-between-js-and-sass